### ISSUE TITLE
Add default values for bgp peers passwords & ports

### ISSUE THREAD

**Issue by donch** [enix/helm-charts#67]
Hi, 

It could be cool to specify or not bgp peer passwords and port. We can put a default value if they are not provided. 
If we don't specify theses values, kube-router refuse to start.

**Comment by gclawes** [comment]
On a related note, the `values.yaml` examples show the port key as `bgpPort`, but the templates expect `port`:
https://github.com/enix/helm-charts/blob/master/charts/kube-router/values.yaml#L123
```
    # kubeRouter.router.peers -- List of external BGP peers, see values.yaml for example
    peers: []
    # - ip: "1.2.3.4"
    #    asn: 65000
    #    password: ""
    #    bgpPort: 179
 ```
 
 https://github.com/enix/helm-charts/blob/master/charts/kube-router/templates/daemonset.yaml#L57
 ```
         - "--peer-router-ports={{ range $i, $peer := .peers }}{{ if not (eq $i 0) }},{{ end }}{{ $peer.port }}{{ end }}"
```

**Comment by npdgm** [comment]
> On a related note, the `values.yaml` examples show the port key as `bgpPort`, but the templates expect `port`

It was fixed months ago in branch `next/kube-router` but never got merged :( I'll make sure it lands on the next release
Thanks

**Comment by npdgm** [comment]
@donch @gclawes, fixed and released in chart version 1.6.0
Thanks for reporting this, settings are now optional where appropriate with sensible defaults and human friendly error messages.
Cheers :rocket:
